scale/recipe/test/test_views.py

```diff
@@ -1667,6 +1667,74 @@ def test_superseded(self):
         self.assertEqual(len(result['jobs']), 1)
         for recipe_job in result['jobs']:
             self.assertFalse(recipe_job['is_original'])
+            
+
+
+
+class TestRecipesPostViewV6(TransactionTestCase):
+    
+    api = 'v6'
+
+    def setUp(self):
+        django.setup()
+
+        self.job_type1 = job_test_utils.create_job_type(name='scale-batch-creator')
+
+        definition = {
+            'version': '1.0',
+            'input_data': [{
+                'media_types': [
+                    'image/x-hdf5-image',
+                ],
+                'type': 'file',
+                'name': 'input_file',
+            }],
+            'jobs': [{
+                'job_type': {
+                    'name': self.job_type1.name,
+                    'version': self.job_type1.version,
+                },
+                'name': 'kml',
+                'recipe_inputs': [{
+                    'job_input': 'input_file',
+                    'recipe_input': 'input_file',
+                }],
+            }],
+        }
+
+        workspace1 = storage_test_utils.create_workspace()
+        file1 = storage_test_utils.create_file(workspace=workspace1)
+
+        data = {
+            'version': '1.0',
+            'input_data': [{
+                'name': 'input_file',
+                'file_id': file1.id,
+            }],
+            'workspace_id': workspace1.id,
+        }
+
+        self.recipe_type = recipe_test_utils.create_recipe_type(name='my-type', definition=definition)
+        recipe_handler = recipe_test_utils.create_recipe_handler(recipe_type=self.recipe_type, data=data)
+        self.recipe1 = recipe_handler.recipe
+        self.recipe1_jobs = recipe_handler.recipe_jobs
+
+        self.recipe2 = recipe_test_utils.create_recipe()
+        self.recipe3 = recipe_test_utils.create_recipe(is_superseded=True)
+
+    def test_successful(self):
+
+        
+        json_data = { 
+            "input" : {},
+            "recipe_type_id" : self.recipe_type.pk
+        }
+
+        url = '/%s/recipes/' % self.api
+        response = self.client.generic('POST', url, json.dumps(json_data), 'application/json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED, response.content)
+
+        #Response should be new v6 job detail response
```

Comment:
Check that location is set in header to point to recipe details page.
Check that data field is not present so we can be sure we have the v6 details serializer